apps/portal/src/components/color-palette.tsx

```diff
@@ -0,0 +1,77 @@
+import * as styles from "@harnessio/core-design-system/styles-esm";
+import { Tooltip } from "@harnessio/ui/components";
+
+interface Color {
+  name?: string;
+  $value?: string;
+  [key: string]: unknown;
+}
+
+export function ColorPalette() {
+  const shades = Object.keys(styles.colors.blue);
+  const colors = Object.keys(styles.colors).filter(
+    (color) => color !== "pure",
+  ); /** Excluding "pure" colors intentionally */
+
+  return (
+    <div className="grid rounded-lg border border-borders-4 bg-background-4 px-6 pt-1 pb-6 mt-4 shadow-md h-fit w-[800px]">
```

Comment:
There's a couple of invalid tokens in here and it looks like ESLint fix wasn't run as well

Reply:
I ran `pnpm lint` in `apps/portal` but not seeing any lint errors.
<img width="826" alt="image" src="https://github.com/user-attachments/assets/de04f4a2-b6aa-436d-aa24-f9d0b7ef66e2" />